### ISSUE TITLE
`pyxdg==0.26` and `xdg==5.1.1` clash

### ISSUE THREAD

**Issue by stdedos** [Xpra-org/xpra#3803]
**Describe the bug**

_It is really a shot in the dark_, but I am trying not to forget it 😅 

Do you "somehow install" on the "OS-namespace the `xdg==5.1.1` package?

I understand that the `pyxdg==0.26` is probably tons old, however, it is so that that's coming via `apt-get install python3-xdg`, and clashes with `xdg==5.1.1` one.

At least in my case, the important stuff, is that it blocks Kazam https://launchpad.net/kazam from working (https://bugs.launchpad.net/kazam/+bug/1956472).

However, even trying to remove it, gives me:

```console
$ sudo apt-get remove python3-xdg
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following packages were automatically installed and are no longer required:
  gir1.2-gudev-1.0 gir1.2-keybinder-3.0 libkeybinder-3.0-0 python3-brlapi python3-gst-1.0 python3-louis python3-pyatspi python3-xlib xbrlapi
Use 'sudo apt autoremove' to remove them.
The following packages will be REMOVED:
  kazam orca python3-speechd python3-xdg
0 upgraded, 0 newly installed, 4 to remove and 1 not upgraded.
After this operation, 6359 kB disk space will be freed.
```

and

```console
$  sudo pip uninstall pyxdg
Found existing installation: pyxdg 0.26
Not uninstalling pyxdg at /usr/lib/python3/dist-packages, outside environment /usr
Can't uninstall 'pyxdg'. No files were found to uninstall.
```

Which "I don't know how sane it is" 😓.

Just an FYI; apologies if it's not you that's doing that.
(but I have to purge `xdg==5.1.1` at least temporarily).

**To Reproduce**
1. server command
2. client command
3. specific action to trigger the bug

**System Information (please complete the following information):**
 - Server OS: [e.g. RHEL 8.x]
 - Client OS: [e.g. Windows 10]
 - Xpra Server Version [e.g. 4.2.3]
 - Xpra Client Version [e.g. 4.3]

**Additional context**
Please see "reporting bugs" in the wiki section.

**Comment by stdedos** [comment]
Nope, not xpra's fault (`autoimport`)

**Comment by totaam** [comment]
@stdedos I don't understand the `autoimport` link - what's the relevance here?
xpra only recommends 2 `xdg` related imports:
https://github.com/Xpra-org/xpra/blob/acfa8cf5ce460950f89caf271875a174547d0354/packaging/rpm/xpra.spec#L349
So you should be able to remove these without affecting xpra.

And `python3-pyxdg` only depends on `python3` and some `rpmlib` bits.

**Comment by stdedos** [comment]
I thought it was xpra that "installed that offending-to-me package", because I found an `import xdg` in the code.

However, since the project "has no `requirements.txt`" (or I don't know its equivalent), and `pyxdg==0.26` is "OS-packaged" (and therefore old), I assumed that "maybe xpra installs/imports a better version".

... My assumptions quickly proved wrong, when I tried to `pip uninstall xdg` the package. However, I had created the issue already 😓
